src/constants/navigation-routes.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
We should avoid grouped default export. We should treat all constants as named export.